src/lib/cimb_translator/Config.cpp

```diff
@@ -1,43 +0,0 @@
-/* This code is subject to the terms of the Mozilla Public License, v.2.0. http://mozilla.org/MPL/2.0/. */
-#include "Config.h"
-#include <cmath>
-
-namespace cimbar {
-
-unsigned Config::total_cells()
```

Comment:
Because these are now per-config, and because we'd like to query some things without actually changing the config (ex: "Is the fountain chunk size different between 4C and B?"), we move all these functions down a layer into the config class.